proto/eth/v2prysm/BUILD.bazel

```diff
@@ -0,0 +1,163 @@
+##############################################################################
+# Common
+##############################################################################
+
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+##############################################################################
+# Go
+##############################################################################
+# gazelle:ignore
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+load("//proto:ssz_proto_library.bzl", "ssz_proto_files")
+load("//tools:ssz.bzl", "SSZ_DEPS", "ssz_gen_marshal")
+
+##############################################################################
+# OpenAPI (Swagger) V2
+##############################################################################
+load("@com_github_grpc_ecosystem_grpc_gateway_v2//protoc-gen-openapiv2:defs.bzl", "protoc_gen_openapiv2")
+
+proto_library(
+    name = "proto",
+    srcs = [
+        ":ssz_proto_files",
+        "sync_committee.proto",
+        "validator.proto",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//proto/eth/ext:proto",
+        "//proto/eth/v1alpha1:proto",
+        "@com_github_grpc_ecosystem_grpc_gateway_v2//protoc-gen-openapiv2/options:options_proto",
+        "@com_google_protobuf//:any_proto",
+        "@com_google_protobuf//:descriptor_proto",
+        "@com_google_protobuf//:empty_proto",
+        "@com_google_protobuf//:timestamp_proto",
+        "@go_googleapis//google/api:annotations_proto",
+    ],
+)
+
+##############################################################################
+# Go
+##############################################################################
+go_library(
+    name = "aliases",
+    srcs = [
+        "aliases.go",
+    ],
+    importpath = "github.com/prysmaticlabs/prysm/proto/eth/v2prysm",
+    visibility = ["//visibility:public"],
+)
+
+ssz_gen_marshal(
+    name = "ssz_generated_files",
+    go_proto = ":go_proto",
+    includes = [
+        "@com_github_prysmaticlabs_eth2_types//:go_default_library",
+        "@com_github_prysmaticlabs_go_bitfield//:go_default_library",
+        "//proto/eth/v1alpha1:go_default_library",
+    ],
+    objs = [
+        # "BeaconBlockAltair",
+        "SyncAggregateAltair",
+        "BeaconBlockAltair",
+        # "SyncCommitteeContribution",
+        # "ContributionAndProof",
+        # "SignedContributionAndProof",
+        # "SignedBeaconBlockAltair",
```

Comment:
Are these comments placeholders for future objects? If yes, then you at least `# "BeaconBlockAltair"` can be removed.